website/screens/overview/releases/ReleasesPage.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  DxcHeading,
+  DxcParagraph,
+  DxcFlex,
+  DxcSpinner,
+  DxcGrid,
+  DxcLink,
+  DxcBulletedList,
+} from "@dxc-technology/halstack-react";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import PageHeading from "@/common/PageHeading";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import HalstackMarkdownParser from "@/common/HalstackMarkdownParser";
+import Link from "next/link";
+import Code from "@/common/Code";
+
+type Section = {
+  title: string;
+  level?: 1 | 2 | 3 | 4 | 5;
+  content?: React.ReactNode;
+  subSections?: Section[];
+  navSubtitle?: string;
+};
+type Release = {
+  name: string;
+  body: string;
+  published_at: string;
+};
+
+const sections = [
+  {
+    title: "Major released versions",
+    content: (
+      <>
+        <DxcParagraph>
+          Every major release of Halstack is collected below:
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/halstack/9/overview/introduction/">
+              Halstack 9
+            </DxcLink>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/halstack/8/overview/introduction/">
+              Halstack 8
+            </DxcLink>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/halstack/7/overview/introduction/">
+              Halstack 7
+            </DxcLink>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/halstack/6/overview/introduction/">
+              Halstack 6
+            </DxcLink>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <DxcParagraph>
+          All the other previous versions, such as Halstack 5 or Halstack 4, are
+          still available on the{" "}
+          <Link
+            href="/overview/introduction/#legacy-documentation-sites"
+            passHref
+          >
+            <DxcLink>legacy sites</DxcLink>
+          </Link>
+          . But remember, these sites are no longer maintained. Consider
+          upgrading if you are currently using one of them.
```

Comment:
I would change the `But remember...` sentence to `Please, take into account that these sites are no longer maintained and consider upgrading if you are currently using one of them.`

Reply:
I thought about this. I decided to write _"But remember..."_ instead of "Please, take into account..." because I'm currently linking to the legacy sites section a sentence before. There, we have that same sentence: 

![image](https://github.com/dxc-technology/halstack-react/assets/44321109/cca37091-e0ba-4125-a761-85d64658926f)

So, to avoid this repetition, I decided to refactor the whole sentence.